src/sci/sci.cc

```diff
@@ -126,6 +126,8 @@ SigmaVectorType SelectedCIMethod::sigma_vector_type() const { return sigma_vecto
 
 size_t SelectedCIMethod::max_memory() const { return max_memory_; }
 
+std::vector<double> SelectedCIMethod::get_PQ_spin2() { return std::vector<double>(); }
```

Comment:
No. This will just return the vector `spin2_` for all derived classes. This is because in all codes spin2 is not computed on demand, but by `compute_energy()`.